make_profiler/lint_makefile.py

```
@@ -2,14 +2,24 @@
 import re
 
 import sys
-from typing import Callable, Dict, List, Set, Tuple
+from typing import Callable
 
 from make_profiler.parser import parse
 import collections
```

Pylint (3.3.7)

[convention] standard import "collections" should be placed before first party import "make_profiler.parser.parse" 

(C0411)